integration_tests/src/db_migration_test.rs

```diff
@@ -41,7 +41,7 @@ pub async fn run_db_migration_test() {
     // Add a bunch of dummy clients to the db to migrate
     add_dummy_clients_to_db(num_clients, &conn);
 
-    thread::sleep(Duration::from_secs(10));
+    thread::sleep(Duration::from_secs(100000));
```

Comment:
is this intentional? that's a long wait?

Reply:
whoops, forgot to change this